ui/client/datasets/DataTransformation/DataTransformation.js

```diff
@@ -664,7 +652,10 @@ const DataTransformation = ({
   };
 
   const processGadmOverrides = () => {
-    transformationsRef.current.overrides = { gadm: savedGADMOverrides };
+    // only add GADM overrides to the transformationsRef if they are present
```

Comment:
We use this ref to determine whether we want to update the annotations object, so if we add this here then we end up updating the annotations object with `overrides: { gadm: null }`, which seems pointless.